native/python3/Makefile

```diff
@@ -29,10 +29,12 @@ PIP_NATIVE = $(WORK_DIR)/../../../native/$(PKG_NAME)/work-native/install/usr/loc
 
 .PHONY: nativepython3_post_install
 nativepython3_post_install: $(WORK_DIR)/python-native.mk
-	@$(MSG) Installing setuptools, pip and cffi
-	@$(RUN) wget https://bootstrap.pypa.io/ez_setup.py -O - | $(PYTHON)
+	@$(MSG) Installing pip, setuptools, cffi and cross env
 	@$(RUN) wget https://bootstrap.pypa.io/get-pip.py -O - | $(PYTHON)
+	@$(PIP) install "setuptools==44.1.0"
 	@$(PIP) install "cffi==1.14.0"
+	@$(PIP) install "crossenv==0.7"
+	@$(PIP) install "cython==0.29.21"
```

Comment:
cython is required to compile some wheels (in my case: `numpy`).
As such I though it would be nice to actually ensure it gets integrated in the native environment.